tests/golden/defaults/floodgate/floodgate/10_deployment.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
```

Comment:
Objects are missing the namespace. I know sometimes™️ argo inserts it automatically. Is it missing on purpose?

Reply:
ArgoCD will deploy all objects without an explicit `metadata.namespace` field to the target namespace configured in the ArgoCD app, so technically, it's not required to specify the namespace in the manifests.